examples/README.md

```diff
@@ -54,6 +54,16 @@ Current examples:
   -advertise-client-urls http://0.0.0.0:2379 \
   -listen-client-urls http://0.0.0.0:2379
   ```
+  **Note for ARM64:**
```

Comment:
These notes for different architecture should have it's own document (perhaps at `docs/ARM64.md`) and just keep short note and link to it.

Reply:
Done.